### ISSUE TITLE
Can i ask of you to make a youtube video about it?

### ISSUE THREAD

**Issue by Aishi-teru** [sleeyax/aternos-discord-bot#3]
honestly i'm new to this. i really dont know where to start. creating a youtube video will help :)) if you could.  Thankyou

**Comment by sleeyax** [comment]
Yes I'm considering making a video for less tech savvy people showing how to get this up and running from 0, I just didn't find the time yet to do so.

Before you ask I've also considered to host a public bot for this so you don't have to host it yourself but the main problem with that is it would need to store your auth cookies and server id in a database. Doing so would be a privacy and security risk to your aternos account (and possibly more if it's linked to an external service e.g a google account). It could also end up being expensive on my end and since it's technically against Aternos TOS all of this combined makes for a bad formula.

**Comment by Aishi-teru** [comment]
Oh. maybe i will wait. cuz i don't know shit about this hahaha. thank you for your response

**Comment by sleeyax** [comment]
No problem. Is there anything in particular you're stuck on though or does the README just not make sense to you at all? I can take your response into account when/if I get to make a vid about it.

---

> Before you ask I've also considered to host a public bot for this so you don't have to host it yourself but the main problem with that is it would need to store your auth cookies and server id in a database. Doing so would be a privacy and security risk to your aternos account (and possibly more if it's linked to an external service e.g a google account). It could also end up being expensive on my end and since it's technically against Aternos TOS all of this combined makes for a bad formula.

And thinking more about this I might just end up adding this functionality to my bot and make it public (and open source) anyways. That way you only have to get the access tokens, set it up once using a discord command and be done with it. Makes it easier for most people I guess (though use would be at own risk and I'm taking my would-be-public bot down if I notice it being abused).

**Comment by Aishi-teru** [comment]
I don't know how to install the whole code. Hehe

**Comment by sleeyax** [comment]
Uhh you don't need to download any code, there's binaries (executables) on the release page on GitHub, as stated in this project's README.

**Comment by Aishi-teru** [comment]
I don't know how to run this. I tried importing the github release to replit.com. i can't run it

**Comment by sleeyax** [comment]
But you don't need to do that. Please, just read https://github.com/sleeyax/aternos-discord-bot/blob/main/README.md#usage it's not that hard.

**Comment by Aishi-teru** [comment]
Can i run this without my pc on? like anytime my friends can just use the bot without me ?

**Comment by sleeyax** [comment]
Yes, that's the whole point actually.

**Comment by Aishi-teru** [comment]
![image](https://user-images.githubusercontent.com/43822412/160901510-091aa451-d840-4525-9149-d83806e714c3.png)


what to do next? i did the steps on the readme

**Comment by sleeyax** [comment]
Oh turns out `go install` doesn't work for the moment due to a replace directive (it's a go limitation for the moment). I created an issue for  this here #5 . In the meantime, download the binary from [releases](https://github.com/sleeyax/aternos-discord-bot/releases) instead.

**Comment by Aishi-teru** [comment]
Still doesn't work :)

**Comment by sleeyax** [comment]
What doesn't work now? You download the executable for your operating system, unzip it, cd into it from your terminal (cmd on windows) and execute the command from the readme. This is very basic stuff. Now I get why I need to make a vid or at least simplify the process for users like you...

**Comment by Aishi-teru** [comment]
![image](https://user-images.githubusercontent.com/43822412/161123121-cd9b18b2-8dd2-4a09-aca1-71e454a8ba4d.png)
i did what exactly on the read.me i understand the unzip file this is what i meant. it stays like that. i left it maybe like a hour? maybe it's loading or something but it doesn't

**Comment by sleeyax** [comment]
Status update: I'm overhauling this bot to support multiple discord servers. That way I (or other ppl too) can host the instance so noobs like you can just add a bot to a server, configure it by copy and pasting literally 2 values and be done with it. Oh, it's also using the newer slash commands now!

<img width="459" alt="image" src="https://user-images.githubusercontent.com/30344294/162339238-fe00161e-c34c-4654-84e2-f84c5c631b98.png">

**ETA is by the weekend or so, I'm almost done.**
